editor.planx.uk/src/@planx/components/shared/PrivateFileUpload/UploadedFileCard.tsx

```diff
@@ -91,74 +92,89 @@ export const UploadedFileCard: React.FC<Props> = ({
   removeFile,
   onChange,
   tags,
+  status,
 }) => (
   <Root>
-    <FileCard>
-      <ProgressBar
-        width={`${Math.min(Math.ceil(progress * 100), 100)}%`}
-        role="progressbar"
-        aria-valuenow={progress * 100 || 0}
-        aria-label={file.path}
-      />
-      <FilePreview>
-        {file instanceof File && file?.type?.includes("image") ? (
-          <ImagePreview file={file} url={url} />
-        ) : (
-          <FileIcon />
-        )}
-      </FilePreview>
-      <Box
-        sx={{ display: "flex", justifyContent: "space-between", width: "100%" }}
-      >
-        <Box mr={2}>
-          <Typography
-            variant="body1"
-            pb="0.25em"
-            sx={{ overflowWrap: "break-word", wordBreak: "break-all" }}
+    <ErrorWrapper
+      error={
+        status === "error"
+          ? "Upload failed, please remove file and try again"
+          : undefined
+      }
+    >
```

Comment:
Adding this wrapper is the change here, the diff below is purely a whitespace change due to the extra indentation but git hasn't quite caught this.